modin/backends/pandas/query_compiler.py

```diff
@@ -354,7 +358,7 @@ def _append_list_of_managers(self, others, axis, **kwargs):
     def _join_list_of_managers(self, others, **kwargs):
         assert isinstance(
             others, list
-        ), "This method is for lists of DataManager objects only"
+        ), "This method is for lists of QueryCompiler objects only"
```

Comment:
Would you be okay to move the doc changes to a new PR so the diff is easier to read for this one?